solr/solr-ref-guide/src/coreadmin-api.adoc

```diff
@@ -118,7 +201,33 @@ http://localhost:8983/solr/admin/cores?action=CREATE&name=my_core&collection=my_
 
 The RELOAD action loads a new core from the configuration of an existing, registered Solr core. While the new core is initializing, the existing one will continue to handle requests. When the new Solr core is ready, it takes over and the old core is unloaded.
 
-`admin/cores?action=RELOAD&core=_core-name_`
+[.dynamic-tabs]
+--
+[example.tab-pane#v1coreadmin-reload]
+====
+[.tab-label]*V1 API*
+
+[source,bash]
+----
+http://localhost:8983/solr/admin/cores?action=RELOAD&core=techproducts
+
+----
+====
+
+[example.tab-pane#v2coreadmin-reload]
+====
+[.tab-label]*V2 API*
+
+[source,bash]
+----
+curl -X POST http://localhost:8983/api/cores/techproducts -H 'Content-Type: application/json' -d'
```

Comment:
[0] I think you're missing a space between `-d` and the single-quote that starts the request-body, which I think would fail the command?